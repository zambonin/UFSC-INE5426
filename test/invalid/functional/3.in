# too many arguments on map's lambda function
int t[10], output[10]
output = map(lambda int x, y -> x + 2, t)
