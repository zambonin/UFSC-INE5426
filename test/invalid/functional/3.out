[Line 6] semantic error: function lambda expects 2 parameters but received 1
[Line 8] semantic error: map lambda expects 1 parameters but received 2
